test/hotspot/jtreg/runtime/MemberName/ResolvedMethodTableHash.java

```diff
@@ -35,72 +39,89 @@
-                .putShort((short) 1)      // methods_count: 1
-                .putShort((short) 0x109)  //   access_flags: public static native
-                .putShort((short) 5)      //   name_index: #5
-                .putShort((short) 6)      //   descriptor_index: #6
-                .putShort((short) 0)      //   attributes_count: 0
-                .putShort((short) 0)      // attributes_count: 0
-                .position();
-    }
+        // Produce a class file with the given name and a single method:
+        //         public static native void m();
+        private int writeClass(byte[] buf, String className) {
+                return ByteBuffer.wrap(buf)
+                                .putInt(0xCAFEBABE)           // magic
+                                .putInt(50)                           // version: 50
+                                .putShort((short) 7)          // constant_pool_count: 7
+                                .put((byte) 7).putShort((short) 2)
+                                .put((byte) 1).putShort((short) className.length()).put(className.getBytes())
+                                .put((byte) 7).putShort((short) 4)
+                                .put((byte) 1).putShort((short) 16).put("java/lang/Object".getBytes())
+                                .put((byte) 1).putShort((short) 1).put("m".getBytes())
+                                .put((byte) 1).putShort((short) 3).put("()V".getBytes())
+                                .putShort((short) 0x21)   // access_flags: public super
+                                .putShort((short) 1)          // this_class: #1
+                                .putShort((short) 3)          // super_class: #3
+                                .putShort((short) 0)          // interfaces_count: 0
+                                .putShort((short) 0)          // fields_count: 0
+                                .putShort((short) 1)          // methods_count: 1
+                                .putShort((short) 0x109)  //   access_flags: public static native
+                                .putShort((short) 5)          //   name_index: #5
+                                .putShort((short) 6)          //   descriptor_index: #6
+                                .putShort((short) 0)          //   attributes_count: 0
+                                .putShort((short) 0)          // attributes_count: 0
+                                .position();
+        }
 
-    public static void main(String[] args) throws Exception {
-        ResolvedMethodTableHash generator = new ResolvedMethodTableHash();
-        List<MethodHandle> handles = new ArrayList<>();
+        public static void main(String[] args) throws Exception {
+
+                ResolvedMethodTableHashTest generator = new ResolvedMethodTableHashTest();
+                List<MethodHandle> handles = new ArrayList<>();
+
+                int count = args.length > 0 ? Integer.parseInt(args[0]) : 200000;
```

Comment:
no need to check args and have default 20000, it is not executed, actually.
might be just hardcode 1000 here?

Reply:
I fixed this to hardcode 1001 (want 1001 iterations to print the message).

Reply:
This was a good suggestion, thank you.